### ISSUE TITLE
Classification for Neural network

### ISSUE THREAD

**Issue by TTitcombe** [TTitcombe/Python-Machine-Learning-Models#7]
Currently only regression implemented (mse). Need to add softmax sigmoid error functions
